### DISCUSSION TITLE
Question about synthetic data assessments

### DISCUSSION THREAD

**Discussion by chrisdburr** [alan-turing-institute/tric-dt#49] [General]
I've been doing some provisional research around synthetic data quality, comparing dimensions for assessment/evaluation to traditional forms of data quality assessment. 

In the following HackMD note, you will find some references to sources and a simple mapping between synthetic data quality <--> data quality:

https://hackmd.io/@cburr/HyfV3ETyA

Two questions:

1. Are there other references or sources that people would like to recommend?
2. How well do the proposed dimensions of synthetic data quality work for DTs in the different domains (i.e. health, natural environment, infrastructure)?

**Comment by BatoolMM** [comment]
@chrisdburr Did you have a chat with Carsten Maple? I believe the field of "Synthetic Data" has not matured enough to establish a governance framework that simultaneously maximises impact. Many of these questions were raised at an event I attended in Warwick (in Feb 2024), organized by Carsten Maple and the Turing (Finance and Economics Programme/Trustworthy Synthetic Data in Practice project) in February. 

I've also given a VERY high-level talk with Rachael — you can find the slides here - some of these questions Rachael also raised it very briefly at the end of the talk with the audience: [https://zenodo.org/records/10561361](https://zenodo.org/records/10561361).
